adhocracy4/polls/static/PollDetail/PollQuestions.jsx

```diff
@@ -347,8 +359,14 @@ class PollQuestions extends React.Component {
                     )
               ))}
               <Alert onClick={() => this.removeAlert()} {...this.state.alert} />
-              {!this.isReadOnly()
+
+              {this.isReadOnly()
```

Comment:
wasn't sure why the false before the true here, didn't seem to have a purpose, unless I missed it?

Reply:
better this way!